TestFoundation/TestNSHTTPCookieStorage.swift

```diff
@@ -241,9 +241,14 @@ class TestNSHTTPCookieStorage: XCTestCase {
         let bundlePath = bundle.bundlePath
         let pathIndex = bundlePath.range(of: "/", options: .backwards)?.lowerBound
         let task = Process()
-        task.launchPath = bundlePath.substring(to: pathIndex!) + "/xdgTestHelper/xdgTestHelper"
+        #if os(macOS)
```

Comment:
Removed for now (tests on mac will crash)
`xdgTestHelper` is not in TestFoundation, so I would create Bundle with URL at path `bundlePath.substring(to: pathIndex!)` and then get executable path?